src/main/webapp/index.html

```diff
@@ -6,10 +6,21 @@
 </head>
 <body>
   <p></p>
+  <a></a>
   <script>
       fetch('/hello', {method: 'GET'}).then(response => response.text()).then((text) => {
           document.querySelector('p').innerText = text;
       });
+
+      fetch('/auth', {method: 'GET'}).then(response => response.json()).then((loggedInInfo) => {
+          let authLinkElement = document.querySelector('a');
```

Comment:
Selecting by the anchor element is dangerous here. You'll probably have more `<a>` elements in your code later. It's better  to add an ID to the element and use it for the selector.